packages/jaeger-ui/src/components/Monitor/ServicesView/operationDetailsTable/__snapshots__/index.test.js.snap

```diff
@@ -77,7 +77,11 @@ exports[`<OperationTableDetails> Table rendered successfully 1`] = `
               Impact
                 
               <Tooltip
-                overlayClassName="impact-tooltip"
+                classNames={
+                  Object {
+                    "root": "\\"impact-tooltip\\"",
```

Comment:
why is there escaping and double quotes, here and in other changes?

Reply:
![image](https://github.com/user-attachments/assets/c4510a8f-2b45-48eb-94c6-5fb3e4f8ffdb)

In this `packages/jaeger-ui/src/components/Monitor/ServicesView/operationDetailsTable/index.tsx` file, I added extra quotes by mistake. Fix pushed in the latest commit.

Reply:
![image](https://github.com/user-attachments/assets/8e024128-4717-4328-aa4f-fa149871a1f7)